docs/firmware/device-sdk/firmware-upgrade/concepts-dfu.md

```diff
@@ -0,0 +1,12 @@
+---
+title: 'Concepts - Firmware Upgrade'

```

Comment:
These are fine to leave for now but would like them to drop the "Concepts - " and be more descriptive, like "Sending logs."

Reply:
More "sections" per service, but smaller pages?